finalPrep/README.md

```diff
@@ -18,37 +18,40 @@ Some things to consider as you refactor include...
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+![Screen Shot 2021-09-21 at 3 32 17 PM](https://user-images.githubusercontent.com/65195952/134247605-70626c3d-4057-4e05-b82f-1650e9ae10fa.png)
 - `Add Week 2 Screenshot Here`
+![Screen Shot 2021-09-21 at 3 32 45 PM](https://user-images.githubusercontent.com/65195952/134247738-3e71b990-fe57-4bcb-aa89-f8a584516e09.png)
 - `Add Week 3 Screenshot Here`
+![Screen Shot 2021-09-21 at 3 32 25 PM](https://user-images.githubusercontent.com/65195952/134247791-d73236f7-baf3-4398-9a1f-0800fa5251e9.png)
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here:[Mentee: Define the Relationship Prep](https://gist.github.com/philalewis/3d048b59ebb94e588ed94bfdd64f9a0f)
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add a link to your gist here:[Forked Pre Teaching Practice exercise](https://gist.github.com/philalewis/c2e695834b9f4eb1113b4277643da67d)
 
 ### Group Work Prep
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [x] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [x] Add a link to your gist here: [Forked DTR Guiding Questions](https://gist.github.com/philalewis/116b4034f627364cc39652f46ade8788)
```

Comment:
It sounds like you have been through a lot, and I am so glad you're pursuing your goals like this. I also appreciate your perspective on growth through direct feedback. Remembering to keep that spirit when you are tired can be a challenge, though! (At least, speaking from my own experience, hah).